### ISSUE TITLE
Bug: Replay scrubber controls frequently become unresponsive after use

### ISSUE THREAD

**Issue by benvinegar** [getsentry/sentry#35546]
If you interact with Replay details for any appreciable amount of time, the scrubber controls basically no-op (you can click them but nothing happens, no state change, etc.).

**Comment by ryan953** [comment]
I tried looking at this, but don't have great repro steps. Laggyness is visible, but the controls are still somewhat responsive and it depends on my laptop/hardware right now.

What I tried:
- It's been proven before that the M1 Pro Macbook renders things quicker than older models. Replays that are laggy when scrubbing around on older hardware are smoother with the M1. I pulled out a 9 year old core-i15 Macbook to test. That machine is laggier than the M1 in a lot of tasks (System Preferences, etc) as well.
